### ISSUE TITLE
Team 1: Implement get_java_strings function

### ISSUE THREAD

**Issue by mariakimheinert** [GatorGauge/GatorGauge#51]
`get_java_strings()` should look in the config.ini file for the "out" parameter, which is the name of the folder where all of the downloaded repos are contained. Then, for every repo in this folder, `get_java_strings()` should extract the contents of all `.java` files as strings and return a list of strings that are the Java programs. `get_java_strings()` should return something that looks like this:
```
java_strings = ["/** A Java program ... }", "/** Another Java program ... }", ""/** Another Java program in a different repo... }" ... ]
```

**Comment by mariakimheinert** [comment]
@bristola how is this feature going?

**Comment by bristola** [comment]
Its done, but on our branch right now.  Dan and I will be merging that into master in a little bit.

**Comment by mariakimheinert** [comment]
Addressed in PR #76
